### DISCUSSION TITLE
Is it possible to use data-domain=127.0.0.1,localhost

### DISCUSSION THREAD

**Discussion by livid** [plausible/analytics#2007] [Anything Else]
In my use case, the webpages to have Plausible snippets are loaded from a local IPFS gateway like this:

http://127.0.0.1:18181/ipns/olivida.eth/

I tried to use the following snippet but it seems data is not collected:

```
<script defer data-domain="olivida.eth.limo,localhost,127.0.0.1" src="https://plausible.io/js/plausible.js"></script>
```

<img width="1301" alt="Screen Shot 2022-07-11 at 2 19 10 PM" src="https://user-images.githubusercontent.com/11427/178360276-c5c46bb6-8062-48d2-a60c-3f4496eed74b.png">

**Comment by RobertJoonas** [comment]
Hi @livid!

The `data-domain` attribute just acts like a unique identifier to determine the dashboard to which you're sending the analytics events. So currently you would be sending events to three different dashboards with the names specified.

In your screenshot I can see a status 200 for our script, so that seems to work. But if data is not collected, then there's probably some conflict with the script. Could you please see the console for any error messages?

Feel free to also reach out to us at hello@plausible.io.

**Reply by livid** [reply]
Hi @RobertJoonas 

There is something like this in console:

<img width="1119" alt="Screen Shot 2022-07-12 at 4 29 22 AM" src="https://user-images.githubusercontent.com/11427/178480270-48652efa-6a97-4a22-ace9-b83f44545036.png">

The script snippet is on an URL like this:

http://127.0.0.1:18182/ipns/olivida.eth/

That is the URL used inside our app, the ENS domain is loaded through a local IPFS gateway, so its host part in the URL is localhost or 127.0.0.1.

**Reply by RobertJoonas** [reply]
I see. Thanks for the screenshot!

Our script is disabled on localhost by default. But we do have an extension that allows you to track localhost too. See more here: https://plausible.io/docs/script-extensions#all-our-script-extensions

**Reply by livid** [reply]
Wow! That's fantastic! I think that is exactly what I was looking for. I will experiment with it in my app now. Thank you so much!!
